src/main/io/gps.c

```diff
@@ -1243,6 +1756,10 @@ static bool writeGpsSolutionNmea(gpsSolutionData_t *sol, const gpsDataNmea_t *da
                 sol->numSat = data->numSat;
                 sol->llh.altCm = data->altitudeCm;
             }
+             navDeltaTimeMs = (tenthsInTenSeconds + data->secondsInTenths - gpsData.lastNavSolTs) % tenthsInTenSeconds;
+             gpsData.lastNavSolTs = data->secondsInTenths;
+             navDeltaTimeMs *= 100;
```

Comment:
```suggestion
             navDeltaTimeMs = (tenthsInTenSeconds + data->secondsInTenths - gpsData.lastNavSolTs) % tenthsInTenSeconds;
             gpsData.lastNavSolTs = data->secondsInTenths;
             navDeltaTimeMs *= 100;
```
```suggestion
             navDeltaTimeMs = ((tenthsInTenSeconds + data->secondsInTenths - gpsData.lastNavSolTs) % tenthsInTenSeconds) * 100;
             gpsData.lastNavSolTs = data->secondsInTenths;
```

It is ugly to have wrong value in variable (deciseconds instead of ms)

Reply:
Yes but the resolution of the NMEA data is only in deciseconds.

Reply:
renamed to deciseconds

Reply:
Will use milliseconds, and re-use data->time, since it is handled differently anyway in the different packets.